PFCalEE/src/DetectorConstruction.cc

```diff
@@ -1370,7 +1700,7 @@ void DetectorConstruction::UpdateCalorSize(){
     if (version_ == v_HGCALEE_v5 || version_ == v_HGCAL_v5 || version_ == v_HGCALEE_v5_gap4 || version_ == v_HGCAL_v5_gap4) m_z0pos = 2990;//3170;
     else if (version_ == v_HGCALEE_v6 || version_ == v_HGCAL_v6 || version_ == v_HGCALEE_v7 || version_ == v_HGCAL_v7 || version_ == v_HGCAL_v7_HF ||version_ == v_HGCALEE_v624 || version_ == v_HGCALEE_v618) m_z0pos = 3070;
     else if (version_ == v_HGCALEE_v8 || version_ == v_HGCAL_v8 || version_==v_HGCALEE_v8_air3 || version_==v_HGCALEE_v8_air4 || version_==v_HGCALEE_v8_Cu || version_==v_HGCALEE_v8_Cu_12) m_z0pos = 2980;
```

Comment:
do we need to add also v9 there ?

Reply:
PS: if so, may need to adjust thermal screen "air" width to fall again on the right z pos for first layer....

Reply:
ok so i added v9 to the if as well and increased the "thermal screen air" by 10mm to compensate the shift